<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDetails</class>
 <widget class="QWidget" name="AccountDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Other Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Phone Office:</string>
        </property>
        <property name="buddy">
         <cstring>phone_office</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="phone_office"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Fa&amp;x:</string>
        </property>
        <property name="buddy">
         <cstring>phone_fax</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="phone_fax"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Other Phone:</string>
        </property>
        <property name="buddy">
         <cstring>phone_alternate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="phone_alternate"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Emp&amp;loyees:</string>
        </property>
        <property name="buddy">
         <cstring>employees</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="employees"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Rating:</string>
        </property>
        <property name="buddy">
         <cstring>rating</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="rating"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>SIC Code:</string>
        </property>
        <property name="buddy">
         <cstring>sic_code</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="sic_code"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Annual Re&amp;venue:</string>
        </property>
        <property name="buddy">
         <cstring>annual_revenue</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="annual_revenue"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Email Address:</string>
        </property>
        <property name="buddy">
         <cstring>email1</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="email1"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>VAT ID:</string>
        </property>
        <property name="buddy">
         <cstring>vat_no_c</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="vat_no_c"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Visma ID:</string>
        </property>
        <property name="buddy">
         <cstring>visma_id_c</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="visma_id_c"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>Accounting:</string>
        </property>
        <property name="buddy">
         <cstring>accounting_c</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="BetterPlainTextEdit" name="accounting_c">
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="billingAddressGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Street:</string>
        </property>
        <property name="buddy">
         <cstring>billing_address_street</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="BetterPlainTextEdit" name="billing_address_street">
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Cit&amp;y:</string>
        </property>
        <property name="buddy">
         <cstring>billing_address_city</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="billing_address_city"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>State:</string>
        </property>
        <property name="buddy">
         <cstring>billing_address_state</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="billing_address_state"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Postal Code:</string>
        </property>
        <property name="buddy">
         <cstring>billing_address_postalcode</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="billing_address_postalcode"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Co&amp;untry:</string>
        </property>
        <property name="buddy">
         <cstring>billing_address_country</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="billing_address_country"/>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Billing Address</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyBillingAddressButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-copy">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="officeAddressGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="topMargin">
       <number>1</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Street:</string>
        </property>
        <property name="buddy">
         <cstring>shipping_address_street</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="BetterPlainTextEdit" name="shipping_address_street">
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>City:</string>
        </property>
        <property name="buddy">
         <cstring>shipping_address_city</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="shipping_address_city"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>State:</string>
        </property>
        <property name="buddy">
         <cstring>shipping_address_state</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="shipping_address_state"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>Postal Code:</string>
        </property>
        <property name="buddy">
         <cstring>shipping_address_postalcode</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="shipping_address_postalcode"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Country:</string>
        </property>
        <property name="buddy">
         <cstring>shipping_address_country</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="shipping_address_country"/>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Office Address</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="copyOfficeAddressButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-copy">
            <normaloff>.</normaloff>.</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="website"/>
        </item>
        <item>
         <widget class="QPushButton" name="visitWebsiteButton">
          <property name="text">
           <string>Visit</string>
          </property>
          <property name="icon">
           <iconset theme="internet-web-browser">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="ticker_symbol"/>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="parent_id"/>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="ownership"/>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="industry"/>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="account_type"/>
      </item>
      <item row="15" column="1">
       <widget class="QComboBox" name="assigned_user_id"/>
      </item>
      <item row="18" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="viewDocumentsButton">
          <property name="text">
           <string>Open</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="manageDocumentsButton">
          <property name="text">
           <string>&amp;Manage Documents</string>
          </property>
          <property name="icon">
           <iconset theme="document-edit">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Notes:</string>
        </property>
       </widget>
      </item>
      <item row="20" column="1">
       <widget class="QPushButton" name="viewNotesButton">
        <property name="text">
         <string>View Notes</string>
        </property>
        <property name="icon">
         <iconset theme="view-pim-notes">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QLabel" name="urllabel">
        <property name="text">
         <string>-</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Documents:</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Assi&amp;gned to:</string>
        </property>
        <property name="buddy">
         <cstring>assigned_user_id</cstring>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>&amp;Type:</string>
        </property>
        <property name="buddy">
         <cstring>account_type</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Industry:</string>
        </property>
        <property name="buddy">
         <cstring>industry</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ownership:</string>
        </property>
        <property name="buddy">
         <cstring>ownership</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Member of:</string>
        </property>
        <property name="buddy">
         <cstring>parent_id</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tic&amp;ker Symbol:</string>
        </property>
        <property name="buddy">
         <cstring>ticker_symbol</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Website:</string>
        </property>
        <property name="buddy">
         <cstring>website</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Name:</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BetterPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">betterplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>website</tabstop>
  <tabstop>ticker_symbol</tabstop>
  <tabstop>parent_id</tabstop>
  <tabstop>ownership</tabstop>
  <tabstop>industry</tabstop>
  <tabstop>account_type</tabstop>
  <tabstop>assigned_user_id</tabstop>
  <tabstop>phone_office</tabstop>
  <tabstop>phone_fax</tabstop>
  <tabstop>phone_alternate</tabstop>
  <tabstop>employees</tabstop>
  <tabstop>rating</tabstop>
  <tabstop>sic_code</tabstop>
  <tabstop>annual_revenue</tabstop>
  <tabstop>email1</tabstop>
  <tabstop>vat_no_c</tabstop>
  <tabstop>visma_id_c</tabstop>
  <tabstop>accounting_c</tabstop>
  <tabstop>billing_address_street</tabstop>
  <tabstop>billing_address_city</tabstop>
  <tabstop>billing_address_state</tabstop>
  <tabstop>billing_address_postalcode</tabstop>
  <tabstop>billing_address_country</tabstop>
  <tabstop>shipping_address_street</tabstop>
  <tabstop>shipping_address_city</tabstop>
  <tabstop>shipping_address_state</tabstop>
  <tabstop>shipping_address_postalcode</tabstop>
  <tabstop>shipping_address_country</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
